hardhat.config.ts

```diff
@@ -0,0 +1,58 @@
+import '@nomicfoundation/hardhat-toolbox';
+import 'hardhat-deploy';
+import { HardhatUserConfig } from 'hardhat/config';
+const { WALLET_PRIVATE_KEY } = process.env;
```

Comment:
```suggestion
import { HardhatUserConfig } from 'hardhat/config';
import '@nomicfoundation/hardhat-toolbox';
import '@nomiclabs/hardhat-etherscan';

const { WALLET_PRIVATE_KEY } = process.env;
```
remove useless library for deploying SC. It's not currently the best method to do it

Reply:
useless to add import '@nomiclabs/hardhat-etherscan'; as it exists in hardhat toolbox